api/src/lists/lists.service.ts

```diff
@@ -90,8 +92,12 @@ export class ListsService {
     return userListDetails;
   }
 
-  async createList(createList: CreateListDto, userId: string) {
-    return await this.listsDao.createList(createList, userId);
+  async createList(movies: CreateListDto, userId: string) {
+    const list = await this.listsDao.createList(movies, userId);
+
+    await this.moviesService.createMovies(movies, list.id);
```

Comment:
how come you can pass `movies` to both? shouldn't be `movies` for `listDao`  and `movies.movie` for `createMovies`?

Reply:
Both were written to take an entire list instead of only the properties they need, should be fixed with the [above review](https://github.com/funnls/cinesync/pull/43#discussion_r1507792837).